src/main/java/seedu/address/logic/parser/UserInputParser.java

```diff
@@ -59,13 +65,17 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public Command parseCommand(String userInput) throws ParseException {
+        //store userInput in inputCommandStorage
+        inputCommandStorage.newInput(userInput);
```

Comment:
when I tested this there are no inputs being added to the inputcommand storage zz fixing this too

Reply:
oh got it!